spec/dor/services/client/collections_spec.rb

```diff
@@ -23,6 +23,7 @@
               "label":"my collection",
               "version":1,
               "description":{
+                "purl":"https://purl.stanford.edu/bc123df4567",
```

Comment:
do collections get a purl page?

Reply:
Clicked on a couple links in Argo and I can confirm they do.